Testing/SystemTests/tests/analysis/WeightedLeastSquaresTest.py

```diff
@@ -45,26 +46,78 @@ def run_fit(wks, function, minimizer):
 
     return params, errors
 
-def get_evs14188_90processed():
+def compare_relative_errors(values_fitted, reference_values, tolerance=1e-6):
     """
-    Load a processed ISIS Vesuvio data. This processed data contains a couple of peaks.
+    Checks that all the values 'fitted' do not differ from reference
+    values by a tolerance/threshold or more. This method compares
+    the relative error. An epsilon of 0.01 means a relative
+    difference of 1 % = 100*0.01 %. If the reference value is 0,
+    checks that the difference from 0 is less than the tolerance.
+
+    @param values_fitted :: parameter values obtained from a calculation or algorithm (fitting for example)
+    @param reference_values :: (expected) reference value
+
+    @returns if val differs in relative terms from ref by less than epsilon
```

Comment:
A suggestion for change is 

```
+    @returns if val differs in relative terms from ref by less than epsilon
```

to

```
+    @returns if val differs in relative terms from ref by less than epsilon. Otherwise Error is raised with description why comparison/test failed 
```